wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageSectionController.swift

```diff
@@ -139,39 +139,31 @@ final class ConversationMessageSectionController: NSObject, ZMMessageObserver {
         isSenderVisible: Bool,
         to cellDescriptions: inout [AnyConversationMessageCellDescription]
     ) {
+        let showEphemeralTimer = message.isEphemeral && !message.isObfuscated
```

Comment:
I moved this line up and injected the value in each of the cell creation functions.